Templates/DefaultComponent/Template/Include/${Name}Interface.h

```diff
@@ -0,0 +1,48 @@
+// {BEGIN_LICENSE}
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+// {END_LICENSE}
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Component/EntityId.h>
+
+namespace ${SanitizedCppName}
+{
+    class ${SanitizedCppName}Requests
+    {
+    public:
+        AZ_RTTI(${SanitizedCppName}Requests, "{${Random_Uuid}}");
```

Comment:
It's recommended to include the namespace in the `AZ_RTTI` macros, especially for components. This helps avoid name conflicts, which will force the Json Serialization to have to use the type id instead of the name.

Reply:
Namespace included now. Thanks!